examples/glue_benchmark/glue.py

```diff
@@ -219,55 +237,69 @@ def preprocess_fn(feature, label):
     test_ds = preprocess_data(preprocess_fn, test_ds)
 
     if FLAGS.load_finetuning_model:
-        finetuning_model = keras.models.load_model(FLAGS.load_finetuning_model)
+        with strategy.scope():
+            finetuning_model = tf.keras.models.load_model(
+                FLAGS.load_finetuning_model
+            )
     else:
-        loss = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
-        metrics = [keras.metrics.SparseCategoricalAccuracy()]
-        if FLAGS.task_name == "stsb":
-            num_classes = 1
-            loss = keras.losses.MeanSquaredError()
-            metrics = [keras.metrics.MeanSquaredError()]
-        elif FLAGS.task_name in (
-            "mnli",
-            "mnli_mismatched",
-            "mnli_matched",
-            "ax",
-        ):
-            num_classes = 3
-        else:
-            num_classes = 2
-
-        # ----- Custom code block starts -----
-        # Users should change this `BertClassifier` to your own classifier.
-        # Commonly the classifier is simply your model + several dense layers,
-        # please refer to "Make the Finetuning Model" section in README for
-        # detailed instructions.
-        bert_model = keras_nlp.models.BertBackbone.from_preset(
-            "bert_tiny_uncased_en"
-        )
-        finetuning_model = keras_nlp.models.BertClassifier(
-            backbone=bert_model,
-            num_classes=num_classes,
-        )
-        # ----- Custom code block ends -----
-
-        finetuning_model.compile(
-            optimizer=keras.optimizers.experimental.AdamW(FLAGS.learning_rate),
-            loss=loss,
-            metrics=metrics,
-        )
+        with strategy.scope():
```

Comment:
Could we just put everything inside the strategy scope? Or does fit need to be outside?

If we can might be easier to pull the scope over the whole block here.